mpw_precheck.py

```diff
@@ -45,14 +46,34 @@ def log_info(precheck_config, project_config):
         logging.info(f"{{{{Tools Info}}}} KLayout: v{klayout_version} | Magic: v{magic_version}")
     with open(pdks_info_path, 'w') as pdks_info:
         try:
-            pdk_dir = f"{precheck_config['pdk_path'].parent}/%s"
-            open_pdks_v_cmd = ['git', '-C', pdk_dir % 'open_pdks', 'rev-parse', '--verify', 'HEAD']
-            skywater_pdk_v_cmd = ['git', '-C', pdk_dir % 'skywater-pdk', 'rev-parse', '--verify', 'HEAD']
-            open_pdks_version = subprocess.check_output(open_pdks_v_cmd, encoding='utf-8').rstrip()
-            skywater_pdk_version = subprocess.check_output(skywater_pdk_v_cmd, encoding='utf-8').rstrip()
-            pdks_info.write(f"Open PDKs {open_pdks_version}\n")
-            pdks_info.write(f"Skywater PDK {skywater_pdk_version}")
-            logging.info(f"{{{{PDKs Info}}}} PDK: {precheck_config['pdk_path'].name} | Open PDKs: {open_pdks_version} | Skywater PDK: {skywater_pdk_version}")
+            volare_node_info = Path.joinpath(precheck_config['pdk_path'], '.config/nodeinfo.json')
```

Comment:
`pathlib` also support things like `volare_node_info = pathlib.Path(precheck_config['pdk_path']) / '.config/nodeinfo.json', which is kind-of cute.